fhir/resources/core/fhirabstractmodel.py

```diff
@@ -9,13 +9,13 @@
 from enum import Enum
 from functools import lru_cache
 
+from pydantic import ValidationError
 from pydantic.v1 import BaseModel, Extra, Field
 from pydantic.v1.class_validators import ROOT_VALIDATOR_CONFIG_KEY, root_validator
-from pydantic.v1.error_wrappers import ErrorWrapper, ValidationError
-from pydantic.v1.errors import ConfigError, PydanticValueError
```

Comment:
These have been removed in pydantic v2 https://docs.pydantic.dev/latest/migration/#removed-in-pydantic-v2